js/index.js

```diff
@@ -12,6 +12,67 @@ const selectRandom = (memberImgArr, n) => {
 
 const numOfMembers = 5;
 let memberSection = document.getElementById('members');
+
+let regexp = /android|iphone|kindle|ipad/i;
+isOpenInPhone(regexp);
+
+export function isOpenInPhone(type) {
+  let details = navigator.userAgent;
+  let isMobileDevice = type.test(details);
+
+  if (isMobileDevice) {
+    openDialog();
+    document.getElementById('okayBt').addEventListener('click', openApp);
+    document.getElementById('cancleBt').addEventListener('click', closeDialog);
+    return true;
+  } else {
+    closeDialog();
+    return false;
+  }
+}
+
+function openDialog() {
+  document.querySelectorAll('.appDialog')[0].style.display = 'block';
+  document.querySelectorAll('.mainDiv')[0].style.display = 'none';
+}
+function closeDialog() {
+  document.querySelectorAll('.appDialog')[0].style.display = 'none';
+  document.querySelectorAll('.mainDiv')[0].style.display = 'block';
+}
+function openRDSApp() {
+  var flag = false;
+  var appScheme = 'app://realdevsquad.com';
+  var fallbackURL =
+    'https://play.google.com/store/apps/details?id=com.github.android'; // It will replace with app playstore url
+
+  var userAgent = navigator.userAgent || navigator.vendor || window.opera;
+  if (/android/i.test(userAgent)) {
+    var iframe = document.createElement('iframe');
+    iframe.style.display = 'none';
+    iframe.src = appScheme;
+    document.body.appendChild(iframe);
+
+    window.addEventListener('blur', function () {
+      document.body.removeChild(iframe);
+      // Calculate the time taken to blur (app was opened)
+      var timeTaken = Date.now() - startTime;
+      if (timeTaken <= 1000) {
+        flag = true;
+      }
+    });
+
+    setTimeout(function () {
+      if (!flag) {
+        document.body.removeChild(iframe);
+        window.location.href = fallbackURL;
+      }
+    }, 1000); // Adjust the delay as needed
+  } else {
```

Comment:
What's the intention of this part of the code?

Reply:
1. This is for if time taken to come from rds mobile app is more then it means app is install.
2. If flag is false means app doesn't install in phone and then it will re-direct to playstore(App mentioned in fallbackurl).